### PR TITLE
Large data Uplink

### PR DESCRIPTION
**Pull request by GVardakis**

### CODE REVIEW

File: satnogsclient/settings.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def _cast_or_none(func, value):
 DEMODULATOR_INIT_TIME = int(environ.get('SATNOGS_DEMODULATOR_INIT_TIME', 5))  # In seconds
 SCHEDULER_SLEEP_TIME = 10  # In seconds
 GNURADIO_UDP_PORT = 16886
-GNURADIO_IP = '127.0.0.1'
+GNURADIO_IP = '147.52.17.78'
```
Comment (surligas) [inline satnogsclient/settings.py]: Fix the IP

**Comment by surligas** [comment]
The uplink now is at 1200 baud, so the timeout between packets should be a little larger. The mechanism works you can proceed with the merge.

**Comment by GVardakis** [comment]
What timeout do you propose?

**Comment by surligas** [comment]
Increase it by +1.5 secs

**Comment by GVardakis** [comment]
Done

**Comment by ppapadeas** [comment]
Rebased and merged

**Pull request closed without merging**